session/session.go

```diff
@@ -712,6 +712,9 @@ func (s *session) SetProcessInfo(sql string) {
 		State:   s.Status(),
 		Info:    sql,
 	}
+	if sql == "" {
```

Comment:
If the query is sent through the binary protocol with prepare-execute method, will the `sql` be an empty string?

Reply:
No, `SetProcessInfo` will use `OriginText` of the execute statement. It is set in `CompileExecutePreparedStmt`.

Reply:
OK